### PR TITLE
Native reference updates

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: docs/src/main/asciidoc/native-reference.adoc [hunk 1]
```diff
@@ -786,8 +775,8 @@ Next, generate a flame graph using the data captured via `perf record`:
 
 [source,bash]
 ----
-$ perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded
-$ ${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
+perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded
+${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
```
Comment (galderz) [inline docs/src/main/asciidoc/native-reference.adoc:779]: For even easier copying, we could convert these two commands into a single one. E.g. 

```
perf script -i perf.data | ${FG_HOME}/stackcollapse-perf.pl > out.perf-folded \ 
    && ${FG_HOME}/flamegraph.pl out.perf-folded > flamegraph.svg
```

**Review by galderz — CHANGES_REQUESTED** [review]
Looks good, just a small comment around combining commands to produce flamegraphs.

File: docs/src/main/asciidoc/native-reference.adoc (hunk 1 above)
Comment (galderz) [reply docs/src/main/asciidoc/native-reference.adoc]: Even better indeed :)

**Review by galderz — APPROVED** [review]

**Comment by gsmet** [comment]
Doesn't apply cleanly to 2.7 and probably not worth the hassle of fixing the conflicts. Skipping.

**Pull request merged**